loadbalancer_public.tf

```diff
@@ -61,6 +61,10 @@ resource "aws_security_group_rule" "lb_public_http" {
   protocol          = "tcp"
   cidr_blocks       = [each.value]
   security_group_id = aws_security_group.quortex_public.id
+
+  lifecycle {
+    create_before_destroy = true
+  }
```

Comment:
Should not it be also on private LBs ?

Reply:
private lb isn't use for production env, if the connection is lose for 1min it's not an issue.